planetary-social/nos
# Update fastlane plugin for sentry and bundler version

## Issues covered
None

## Description
There's a message in the [logs](https://github.com/planetary-social/nos/actions/runs/10100747038/job/27932798302) from our latest build that says:

```
+----------------------------------------+
|        Plugin updates available        |
+--------+--------------+----------------+
| Plugin | Your Version | Latest Version |
+--------+--------------+----------------+
| sentry | 1.17.0       | 1.24.0         |
+--------+--------------+----------------+
[20:29:39]: To update all plugins, just run
[20:29:39]: $ bundle exec fastlane update_plugins
```

So after installing [mise](https://mise.jdx.dev) and the latest version of ruby, I was able to run `update_plugins`. Then it told me to run `bundle update fastlane` to update fastlane, so I did.

## How to test
I'm not sure. This is the fastlane sentry plugin, so I imagine you'd have to deploy and then see if data still appears in Sentry. So, I guess:

1. Deploy a build
2. Make sure Sentry still works
